docs/UsersGuide.asciidoc

```diff
@@ -628,9 +628,9 @@ parameters.
 [horizontal]
 .The following scheduling parameters exist
 
-_NO_OBSOLETE:: Do not obsolete jobs in older builds with same DISTRI and VERSION
-(as is the default behavior). With this option jobs which are currently pending,
-for example scheduled or running, are not cancelled when a new medium is triggered.
+_OBSOLETE:: Obsolete jobs in older builds with same DISTRI and VERSION
+(The default behavior is not obsoleting). With this option jobs which are currently pending,
+for example scheduled or running, are cancelled when a new medium is triggered.
 
 _DEPRIORITIZEBUILD:: Setting this switch '1' will not immediately obsolete jobs of old
```

Comment:
Needs an update as well because the default would not be to obsolete

Reply:
Sorry, I did not understand what `needs an update as well` means. Can you explain more detail?

Reply:
Currently the text says "Setting this […] will not immediately obsolete builds". It is written like this because obsoletion was the default behaviour. But when it is default to not obsolete I suggest to write the text differently, e.g. "Setting this switch to '1' will deprioritize the unfinished jobs of old builds and obsolete them if a configurable limit of priority is reached."

Reply:
I changed it to "Setting this switch '1' will deprioritize the jobs of old builds to a configurable limit of priority."  Is it okay?

Reply:
well, I think we should still mention what happens when the jobs hit that limit. This is what was implied previously by the word "immediately". With my suggestion I wanted to make that more explicit.

Reply:
well, I think we should still mention what happens when the jobs hit that limit. This is what was implied previously by the word "immediately". With my suggestion I wanted to make that more explicit.

Reply:
How about 'Setting this switch to '1' will deprioritize the unfinished jobs of old builds. If the priority reaches the configurable limit which is specified by _DEPRIORITIZE_LIMIT, the unfinished jobs of old builds will be obsoleted.' ?